### ISSUE TITLE
New version feedstock update order

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/vulkan-headers-feedstock#9]
### Comment:

I believe releasing a new version of vulkan starts here. However, there are a few linked repositories that must be released in lock-step

Once the headers are released, the following repos should be rebuilt:

- [vulkan-loader](https://github.com/conda-forge/vulkan-loader-feedstock)
- [vulkan-tools](https://github.com/conda-forge/vulkan-tools-feedstock)
- more to come

I'm not sure if there an easier way to do these kind of lockstep updates.

**Comment by traversaro** [comment]
> I'm not sure if there an easier way to do these kind of lockstep updates.

Perhaps it is a bit late for this, but I wonder if with a bit of insight it could make more sense to have all the main vulkan package in a single feedstock (even if they are part of different repos), so that they could be bumped together. The main rationale is that even if they are different repos, they are always tagged together.

**Comment by hmaarrfk** [comment]
With the unmaintainabikity of qt and pytorch I'm very scared of mega recipes. 

It also makes it very hard for new contributors to help.....

**Comment by JarrettSJohnson** [comment]
Tangentially related: are "meta" recipes something that conda-forge usually entertains? It was something I was going to suggest once I was done with a couple of more vulkan-related packages. If there was something like a `vulkan-sdk` package that would be versioned to install all of the packages of the same version number, that'd improve ergonomics for vulkan devs.

Also 1.3.296 recently came out. I was intending on updating all of the relevant packages.

**Comment by traversaro** [comment]
> Tangentially related: are "meta" recipes something that conda-forge usually entertains? It was something I was going to suggest once I was done with a couple of more vulkan-related packages. If there was something like a `vulkan-sdk` package that would be versioned to install all of the packages of the same version number, that'd improve ergonomics for vulkan devs.

Sorry, I had missed that message. Sure, that is something that can be added, see for example `cuda-version` for selecting the cuda version: https://github.com/search?q=org%3Aconda-forge%20cuda-version&type=code .

**Comment by JarrettSJohnson** [comment]
Here's a convenient list of components that are shipped with the Vulkan SDK mapped to their conda-forge feedstock. Saves me (and hopefully others) from searching around.

[KhronosGroup/glslang](https://github.com/KhronosGroup/glslang) | [feedstock](https://github.com/conda-forge/glslang-feedstock)
[KhronosGroup/SPIRV-Cross](https://github.com/KhronosGroup/SPIRV-Cross) | No feedstock
[SPIRV-Reflect](https://github.com/KhronosGroup/SPIRV-Reflect) | No feedstock
[shader-slang](https://github.com/shader-slang/slang) | [feedstock ](https://github.com/conda-forge/staged-recipes/pull/30878) is in `staged-recipes`--blocked by another issue: 
[KhronosGroup/SPIRV-Tools](https://github.com/KhronosGroup/SPIRV-Tools) | [feedstock](https://github.com/conda-forge/spirv-tools-feedstock)
[KhronosGroup/Vulkan-Headers](https://github.com/KhronosGroup/Vulkan-Headers) | [feedstock](https://github.com/conda-forge/vulkan-headers-feedstock) 
[KhronosGroup/Vulkan-Loader](https://github.com/KhronosGroup/Vulkan-Loader) | [feedstock](https://github.com/conda-forge/vulkan-loader-feedstock)
[KhronosGroup/Vulkan-Tools](https://github.com/KhronosGroup/Vulkan-Tools) | [feedstock](https://github.com/conda-forge/vulkan-tools-feedstock)
[KhronosGroup/Vulkan-ValidationLayers](https://github.com/KhronosGroup/Vulkan-ValidationLayers) | [feedstock](https://github.com/conda-forge/vulkan-validation-layers-feedstock)
[KhronosGroup/Vulkan-Profiles](https://github.com/KhronosGroup/Vulkan-Profiles) | No feedstock
[KhronosGroup/Vulkan-ExtensionLayer](https://github.com/KhronosGroup/Vulkan-ExtensionLayer) | No feedstock
[KhronosGroup/Vulkan-Utility-Libraries](https://github.com/KhronosGroup/Vulkan-Utility-Libraries) | No feedstock
[LunarG/gfxreconstruct](https://github.com/LunarG/gfxreconstruct) | No feedstock
[LunarG/VulkanTools](https://github.com/LunarG/VulkanTools) | No feedstock (These are LunarG's tools)
[LunarG/CrashDiagnosticLayer](https://github.com/LunarG/CrashDiagnosticLayer) | No feedstock
[volk](https://github.com/zeux/volk) | [feedstock](https://github.com/conda-forge/vulkan-volk-feedstock)
[AMD Memory Allocator](https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator) | [feedstock](https://github.com/conda-forge/vulkan-memory-allocator-feedstock)
[microsoft/DirectXShaderCompiler](https://github.com/microsoft/DirectXShaderCompiler) | No feedstock
[google/shaderc](https://github.com/google/shaderc) | [feedstock](https://github.com/conda-forge/shaderc-feedstock)
[SaschaWillems/VulkanCapsViewer](https://github.com/SaschaWillems/VulkanCapsViewer) | No feedstock
[sdl](https://github.com/libsdl-org/SDL) | [feedstock](https://github.com/conda-forge/sdl-feedstock)
[sdl3](https://github.com/libsdl-org/SDL) | [feedstock](https://github.com/conda-forge/sdl-feedstock)
[glm](https://github.com/g-truc/glm) | [feedstock](https://github.com/conda-forge/glm-feedstock)
[MoltenVK](https://github.com/KhronosGroup/MoltenVK) | No feedstock
KosmicKrisp | No feedstock ([currently in Mesa's MR](https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/37522), but currently I am working on updating its opencl feedstock deps)
